hcs-sxc-java-plugins/hcs-sxc-java-plugins-cryptography/src/main/java/com/hedera/hcs/sxc/plugin/cryptography/keyrotation/KeyRotation.java

```diff
@@ -1,4 +1,4 @@
-package com.hedera.hcs.sxc.cryptography;
+package com.hedera.hcs.sxc.plugin.cryptography.keyrotation;
```

Comment:
Can you rename instances of `alice` to `initiator` and `bob` to `responder` ?

Reply:
done